<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopluStokYukleDialog</class>
 <widget class="QDialog" name="TopluStokYukleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>380</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Monospace</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Toplu Stok Kartı Yükle</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>140</y>
     <width>431</width>
     <height>27</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="SecpushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>JSON/CSV Dosya Seç</string>
   </property>
  </widget>
  <widget class="QLabel" name="Bilgilabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>173</y>
     <width>431</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>80</y>
     <width>58</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Dosya:</string>
   </property>
  </widget>
  <widget class="QLabel" name="DosyaAdilabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="YuklepushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(38, 162, 105);</string>
   </property>
   <property name="text">
    <string>Yükle</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="AyarlargroupBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>199</y>
     <width>431</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Aktarım Ayarları</string>
   </property>
   <widget class="QCheckBox" name="fiyatcheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fiyatları es geç</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="miktarcheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>122</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Miktarları</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Adet/KG gir</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxMiktar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="maximum">
     <double>99999999.989999994635582</double>
    </property>
    <property name="value">
     <double>1000.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="KapatpushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(224, 27, 36);</string>
   </property>
   <property name="text">
    <string>Kapat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CsvDosyaCikartButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CSV şablon çıkart</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="kontrolEderekcheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Kontrol ederek yükle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sunucudanpushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Git sunucusundan çeker</string>
   </property>
   <property name="text">
    <string>Sunucudan çek</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CSVyeAktarpushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Kartları CSV'ye aktar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
